R/db-postgres.r

```diff
@@ -67,6 +67,38 @@ sql_translate_env.PostgreSQLConnection <- function(con) {
       },
       str_detect  = function(string, pattern){
         sql_expr(strpos(!!string, !!pattern) > 0L)
+      },
+      # lubridate functions
+      wday = function(x, label=FALSE, abbr = TRUE
```

Comment:
Can you please apply standard tidyverse style here? e.g. https://style.tidyverse.org